packages/kernel-5.15/config-bottlerocket-metal

```diff
@@ -42,6 +77,23 @@ CONFIG_NET_VENDOR_MELLANOX=y
 CONFIG_MLX5_CORE_EN=y
 CONFIG_NET_SWITCHDEV=y
 
+# Myricom network support
+CONFIG_NET_VENDOR_MYRI=y
+CONFIG_MYRI10GE=m
+CONFIG_MYRI10GE_DCA=y
+
+# Pensando network support
+CONFIG_NET_VENDOR_PENSANDO=y
+CONFIG_IONIC=m
+
+# Solarflare network support
+CONFIG_NET_VENDOR_SOLARFLARE=y
+CONFIG_SFC=m
+CONFIG_SRIOV=y
```

Comment:
`CONFIG_SFC_SRIOV`?

Reply:
Ouch, this is similarly embarrassing... I thought I had worked out all these kinks. Fixed in the force push below.